pkg/util/common.go

```diff
@@ -199,3 +199,92 @@ func TagStringToSemver(tag string) (semver.Version, error) {
 func SemverToTagString(tag semver.Version) string {
 	return AddTagPrefix(tag.String())
 }
+
+// CopyFile copies a file from one location to another.
+func CopyFile(src, dst string, ignoreNotExist bool) error {
+	srcStat, err := os.Stat(src)
+	if err != nil && !ignoreNotExist {
+		return err
+	}
+	if os.IsNotExist(err) && ignoreNotExist {
+		return nil
+	}
+
+	if !srcStat.Mode().IsRegular() {
+		return errors.New("file mode must be regular to be copied")
+	}
+
+	source, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer source.Close()
+
+	destination, err := os.Create(dst)
+	if err != nil {
+		return err
+	}
+	defer destination.Close()
+	_, err = io.Copy(destination, source)
+	return err
+}
+
+// CopyDirContents copies directory contents from one location to another.
+func CopyDirContents(src, dst string) error {
+	// If initial destination does not exist create it.
+	if _, err := os.Stat(dst); err != nil {
+		if err := os.MkdirAll(dst, os.FileMode(0755)); err != nil {
```

Comment:
Let’s wrap the above `err` that in case of an issue no information gets lost. We could also wrap with some additional message because those `os` errors are not very verbose per default. :)